src/main/java/lotto/domain/MatchResult.java

```diff
@@ -1,42 +1,37 @@
 package lotto.domain;
 
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 public class MatchResult {
 
     private Map<Rank, Integer> matchResult;
 
     public MatchResult(){
-        matchResult = new HashMap<>();
-        for (Rank rank : Rank.values()){
-            matchResult.put(rank, 0);
-        }
+        matchResult = new HashMap<>(){{
+            put(Rank.FIRST, 0);
+            put(Rank.SECOND, 0);
+            put(Rank.THIRD, 0);
+            put(Rank.FOURTH, 0);
+            put(Rank.FIFTH, 0);
+            put(Rank.MISS, 0);
+        }};
```

Comment:
```suggestion
        this(new HashMap<>(){{
            put(Rank.FIRST, 0);
            put(Rank.SECOND, 0);
            put(Rank.THIRD, 0);
            put(Rank.FOURTH, 0);
            put(Rank.FIFTH, 0);
            put(Rank.MISS, 0);
        }});
```

필드 값 초기화는 주 생성자에서만 하는 것을 추천